config/plugins/interactive_environments/jupyter/config/jupyter.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE interactive_environment SYSTEM "../../interactive_environments.dtd">
+<interactive_environment name="Jupyter (Programming Environment)">
```

Comment:
It's fine IMO. I don't know if there's anything better than Jupyter. Either they know what it is and they launch it intentionally, or they don't know and launch it unintentionally and are confused no matter what the label is.